react/features/notifications/functions.js

```diff
@@ -11,11 +9,9 @@ declare var interfaceConfig: Object;
  * @param {Object|Function} stateful - The redux store state.
  * @returns {boolean}
  */
-export function areThereNotifications(stateful: Object | Function) {
-    const state = toState(stateful);
-    const { enabled, notifications } = state['features/notifications'];
+export function areThereNotifications() {
 
-    return enabled && notifications.length > 0;
+    return false;
```

Comment:
My suggestion is to temporary deactivate without changing of signature this feature like this:
`return false && enabled && notifications.length > 0;`

Reply:
For what reason you want to deactivate this temporary and have the `return false && enabled && notifications.length > 0;`?